src/hardware/serialport/softmodem.cpp

```diff
@@ -33,22 +33,40 @@
 //#include "mixer.h"
 
 
-CSerialModem::CSerialModem(Bitu id, CommandLine* cmd):CSerial(id, cmd) {
-	InstallationSuccessful=false;
-	connected=false;
+CSerialModem::CSerialModem(Bitu id, CommandLine* cmd)
+	:CSerial(id, cmd),
```

Comment:
(Discussion, I think the code is fine in here)

When initializer list is formatted the same way as constructor body it makes it a bit hard to distinguish visually where one ends and other begins. How about using 2 tabs + 2 spaces of indentation for initializer list? As far as I can tell, each piece of DOSBox code is formatted a little bit different when it comes to constructors :(

2 tabs + 2 spaces would look like this:

(1)
```
Aaaa:Aaaa(int x, int y)
		: Bbb(),
		  foo(x),
		  bar(y) {
	for (auto &i : member_tab)
		i = 42;
}
```
If we agreed to switching to using kernel/K&R - like style for opening braces for functions, then it would look like this:

(2)
```
Aaaa:Aaaa(int x, int y)
		: Bbb(),
		  foo(x),
		  bar(y)
{
	for (auto &i : member_tab)
		i = 42;
}
```
IIRC correctly, Google style mandates 6 spaces, but it is in a context of indentation of 2 spaces. Translated to DOSBox it could be 1 tab + 2 spaces, and look like this:

(3)
```
Aaaa:Aaaa(int x, int y)
	: Bbb(),
	  foo(x),
	  bar(y)
{
	for (auto &i : member_tab)
		i = 42;
}
```

And one more option - in the past, I worked in a project that mandated Java-like opening `{` brace for constructors. To make it more readable (and to make it easier to add/remove member fields from classes) we used this wacky style (it looks weird on the surface but makes clear visual distinction between initializer and a body):

(4)
```
Aaaa:Aaaa(int x, int y)
		: Bbb()
		, foo(x)
		, bar(y) {
	for (auto &i : member_tab)
		i = 42;
}

Aaaa:Aaaa(int x, int y)
		: Bbb()
		, bar(y) {}
```

Which format would you prefer? My choice would be (3) or (2).

Reply:
I also like 3; it's easy to remember. 
I will reformat softmodem accordingly.

Reply:
Fixed in f5319290